xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/webjar/liveDataSource.js

```diff
@@ -74,8 +93,17 @@ define('xwiki-livedata-source', ['module', 'jquery'], function(module, $) {
   };
 
   var addEntry = function(source, entry) {
-    return Promise.resolve($.post(getEntriesURL(source), entry).then(entry => entry.values));
+    return Promise.resolve($.post(getEntriesURL(source), entry).then(e => e.values));
```

Comment:
What was wrong with the old code?

Reply:
The lambda parameter `entry` shadows the `addEntry` method `entry` parameter.